### PR TITLE
Snehapar/1583653 add dotnet7 preview7 sdks

### PR DESCRIPTION
**Pull request by snehapar9**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1583653
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
         [InlineData(NetCoreApp21WebApp, "2.1")]
         [InlineData(NetCoreApp31MvcApp, "3.1")]
         [InlineData(NetCoreApp50MvcApp, "5.0")]
-        [InlineData(NetCore7PreviewMvcApp, "7.0.0-preview.6.22324.4")]
+        [InlineData(NetCore7PreviewMvcApp, "7.0.0-preview.7.22375.6")]
```
Comment (pauld-msft) [inline tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs:44]: I know it wasn't like it before, but replacing `7.0.0-preview.7.22375.6` with `DotNetCoreSdkVersions.DotNet70SdkVersion` will keep us from needing to update this in the future to a new hardcoded value

**Comment by pauld-msft** [comment]
This will have conflicts with https://github.com/microsoft/Oryx/pull/1484 that we may want to think about. Specifically, if this is merged in then I will need to update that PR to propagate the versionsToBuild changes into each flavor specific file and then manually publish the bullseye SDKs to dev and then prod.

On the other hand, if that PR is merged first, then this PR will only need to propagate the versionsToBuild changes into each flavor specific file.

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs (hunk 1 above)
Comment (pauld-msft) [reply tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs:44]: Other than that, LGTM :)

**Review by cormacpayne — APPROVED** [review]
:shipit:

**Pull request merged**